dashboard/app/serializers/api/v1/pd/fit_cohort_view_serializer.rb

```diff
@@ -47,12 +48,16 @@ def registered_fit
     object.try(:registered_fit_workshop?)
   end
 
+  def registered_fit_submission_time
+    object.try(FIT_WEEKEND_REGISTRATION_SYMBOL).try(:created_at)
+  end
```

Comment:
it would be great if we could make this a friendlier format to read, like we do in the timestamp log.

Reply:
Do you think we should do it at the server side or client side? Should it be in UTC or local time zone?

Reply:
fixed